pricing-page-exercise/onder/index.html

```diff
@@ -4,10 +4,114 @@
+                    <p class="small">© 2017-2018</p>
```

Comment:
Using a `span` element would be more semantically correct here as the text "© 2017-2018"  is not a paragraph or even a sentence.